src/styles/stitches.config.ts

```diff
@@ -54,6 +63,20 @@ export const { styled, css, globalCss, keyframes, getCssText, theme, createTheme
         paddingX: (value: string) => ({ paddingLeft: value, paddingRight: value }),
         paddingY: (value: string) => ({ paddingTop: value, paddingBottom: value }),
         boxSizingForAllChildren: (value: string) => ({ '*': { boxSizing: value } }),
+        elevationBoxShadow: (value: TElevationType) => {
+            if (!Array.isArray(elevation_box_shadow[`${value}`])) {
```

Comment:
```suggestion
            const box_shadow = elevation_box_shadow[value];
            if (box_shadow) {
```
This check is weird. Shouldn't we check if the property exist?

Reply:
I will check if property exists but we need this check too as the value that comes in design token can be an array or object depending on the number of box shadows, if there is only one box shadow then it is an object ot elese it is an array